cfme/tests/automate/test_git_import.py

```diff
@@ -491,3 +491,37 @@ def check_verify_ssl():
     check_verify_ssl()
     imported_domain.rest_api_entity.action.refresh_from_source()
     check_verify_ssl()
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1716443])
+def test_git_refresh_with_renamed_yaml(appliance):
+    """
+    Note: In this test case, we are checking that datastore with broken __domain__.yml should not
+    get imported. But this BZ says, automate namespace, class, instance, method under domain should
+    be refreshed even if you broke the __domain__.yml once it imported(This could be only tested
+    manually)
+
+    Bugzilla:
+        1716443
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        startsin: 5.10
+        casecomponent: Automate
+        setup:
+            1. Have a git backed Automate Domain
+            2. Delete (or rename) a .rb/.yaml pair, commit, push to repo
+        testSteps:
+            1. Import datastore via git
+        expectedResults:
+            1. Domain should not get imported
+    """
+    repo = appliance.collections.automate_import_exports.instantiate(
+        import_type="git", url="https://github.com/ganeshhubale/ManageIQ-automate-git.git"
```

Comment:
@ganeshhubale now, we have `https://github.com/ManageIQ-automate-git.git`
is it ok to wait and start point from this PR only?

Reply:
Created PR for this test case: https://github.com/RedHatQE/ManageIQ-automate-git/pull/5

Reply:
@mshriver Need more branches named: **without-top-level**, **multi-domains** [in this repo](https://github.com/RedHatQE/ManageIQ-automate-git.git)
- This will help me to make PR with required things against these branches. So that, I can use this repo for all test cases and we can fetch this from cfme_data.yml

Reply:
done

Reply:
Thank you @mshriver 
I will make PRs against these branches and will notify you.

Reply:
I made required PRs. Please have a look:
https://github.com/RedHatQE/ManageIQ-automate-git/pulls